packages/dds/tree/src/simple-tree/objectNodeTypes.ts

```diff
@@ -18,24 +18,27 @@ import type { FieldKey } from "../core/index.js";
  * @privateRemarks
  * This is a candidate for being promoted to the public package API.
  */
-export interface ObjectNodeSchema<
+export type ObjectNodeSchema<
```

Comment:
Curious, why the change from interface to type?

Reply:
This may just be a remnant of an earlier iteration of these changes. I'll confirm.

Reply:
Yes, no longer needed. Reverted.